components/automate-deployment/pkg/assets/binds.go

```diff
@@ -24,14 +24,16 @@ automate-cds REQUIRED compliance-service
 automate-cds BINDING_MODE strict
 automate-cs-bookshelf REQUIRED automate-pg-gateway pg-sidecar-service
 automate-cs-bookshelf BINDING_MODE strict
-automate-cs-nginx REQUIRED automate-cs-bookshelf automate-cs-oc-erchef automate-es-gateway
+automate-cs-nginx REQUIRED automate-cs-bookshelf automate-cs-oc-erchef automate-cs-ocid automate-es-gateway
 automate-cs-nginx OPTIONAL automate-gateway
 automate-cs-nginx BINDING_MODE relaxed
 automate-cs-oc-bifrost REQUIRED automate-pg-gateway pg-sidecar-service
 automate-cs-oc-bifrost BINDING_MODE strict
 automate-cs-oc-erchef REQUIRED automate-cs-bookshelf automate-cs-oc-bifrost automate-es-gateway automate-pg-gateway pg-sidecar-service
 automate-cs-oc-erchef OPTIONAL automate-gateway
 automate-cs-oc-erchef BINDING_MODE relaxed
+automate-cs-ocid REQUIRED automate-pg-gateway pg-sidecar-service
+automate-cs-ocid BINDING_MODE strict
```

Comment:
need clarity on this. can discuss.

Reply:
Need clarity from automate team.

Reply:
@YashviJain01 @daveaugustus Can you please explain these modes to us? If you can point us to any documentation that'd be great.

Reply:
After discussion with Sheik we learnt that this needs to be reverted and it will auto generated by `// Generated by agg_bindings.sh.`

Reply:
running agg_bindings.sh creates the same changes